test/sanity/ignore.txt

```diff
@@ -5000,6 +5000,8 @@ lib/ansible/modules/windows/win_credential.ps1 pslint:PSCustomUseLiteralPath
 lib/ansible/modules/windows/win_credential.ps1 validate-modules:E337
 lib/ansible/modules/windows/win_credential.py future-import-boilerplate
 lib/ansible/modules/windows/win_credential.py metaclass-boilerplate
+lib/ansible/modules/windows/win_data_deduplication.py future-import-boilerplate
```

Comment:
Why is this needed? seems like you should write the docs like written in https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/future-import-boilerplate.html

Reply:
@ShachafGoldstein I based my Python doc on another windows module, and I've see that the documentation for all the windows module have this enabled (otherwise the build of the doc fails)

Reply:
My bad, I thought this was the ignore.txt for the pslint for some reason

Reply:
These ignore entries should no longer be needed -- but you'll need to rebase your PR to pull in the changes and resolve the merge conflicts.